src/plugins/intel_gpu/src/graph/reshape.cpp

```diff
@@ -149,13 +149,18 @@ void reshape_inst::update_shape() {
     auto& node = const_cast<reshape_node&>(dynamic_cast<const reshape_node&>(_node));
 
     if (_node.get_dependencies().size() == 2) {
-        auto shape_mem = _network.get_output_memory(_node.get_dependency(1).id());
+        auto in_node = _node.get_dependency(1).id();
```

Comment:
nit: how about changing to `in_node_id` for code readability?

Reply:
I will change it in next commit. thanks.